changelog/b11997.dd

```diff
@@ -0,0 +1,7 @@
+Check for a D compiler next to rdmd first
+
+Rdmd will now try to use the D compiler it's installed with
+first, whereas before, it would default to invoking a compiler
+from the user's path (which it still falls back to if there's
```

Comment:
"path" should be in uppercase and backticks, so that it's clear the the word refers to the name of an environment variable, and not some nonspecific user path.